docs/source/_templates/layout.html

```diff
@@ -21,6 +21,9 @@
                 <li>
                     <a href="https://optuna.org/#paper" class="header_link">Paper</a>
                 </li>
+                <li>
+                    <a href="https://optuna-zh-cn.readthedocs.io/" class="header_link">中文文档</a>
+                </li>
```

Comment:
I think we aligned the navbar of `ReadTheDocs` with that of https://optuna.org. How about changing the link text to `Community`?

The nav bar of this PR
![image](https://user-images.githubusercontent.com/3255979/99366215-a4b28f80-28fb-11eb-98c8-a880b3ca6981.png)

The nav bar of https://optuna.org
![image](https://user-images.githubusercontent.com/3255979/99366249-b431d880-28fb-11eb-816e-c9fff3b93816.png)



```suggestion
                <li>
                    <a href="https://optuna.org/#community" class="header_link">Community</a>
                </li>
```